nodes/Postiz/Postiz.node.ts

```
@@ -743,6 +763,24 @@ export class Postiz implements INodeType {
 					responseData = await postizApiRequest.call(this, 'POST', '/upload', formData);
 				}
 
+				if (operation === 'uploadFileFromURL') {
+					const url = this.getNodeParameter('url', i) as string;
+					// Validate URL format
+					try {
+						const parsedUrl = new URL(url);
+						if (parsedUrl.protocol !== 'http:' && parsedUrl.protocol !== 'https:') {
+							throw new Error('Invalid protocol');
```

ESLint

[error] Use `ApplicationError`, NodeApiError`, `NodeOperationError`, or `TriggerCloseError` [non-autofixable]

(n8n-nodes-base/node-execute-block-wrong-error-thrown)